src/core/generator/from_openapi.rs

```diff
@@ -0,0 +1,84 @@
+use std::collections::BTreeMap;
+
+use oas3::Spec;
+
+use crate::core::config::{Config, Enum, RootSchema, Type, Union, Upstream};
+
+#[derive(Default)]
+pub struct OpenApiToConfigConverter {
+    pub query: String,
+    pub spec: Spec,
+    pub types: BTreeMap<String, Type>,
+    pub unions: BTreeMap<String, Union>,
+    pub enums: BTreeMap<String, Enum>,
+}
```

Comment:
```suggestion
    config: Config
```
Config holds the same thing. Let's store everything in Config.

Reply:
done